ui/cypress/e2e/dataset_publish.e2e.cy.js

```diff
@@ -60,96 +60,103 @@ const genTransformPairs = (dataset_id) => {
 
 import { waitForElwood } from '../support/helpers';
 
+const username = Cypress.env('DOJO_DEMO_USER');
+const password = Cypress.env('DOJO_DEMO_PASS');
+const hasAuth = Boolean(username) && Boolean(password);
+const auth = hasAuth ? {username, password} : undefined;
+
+const isAuthEnabledRemote = hasAuth;
+
 
 describe('Dataset Register: Publish E2E', { browser: ['chrome', 'chromium', 'firefox'] }, () => {
 
   let dataset_id;
 
   afterEach(() => {
-    console.log('Cleaning seeded artifacts.');
-    cy.task('seed:clean', {type: 'dataset', id: dataset_id});
-    cy.task('seed:clean', {type: 'annotation', id: dataset_id});
-
+    if (!isAuthEnabledRemote) {
+      cy.log('Cleaning seeded artifacts.');
+      cy.task('seed:clean', {type: 'dataset', id: dataset_id});
+      cy.task('seed:clean', {type: 'annotation', id: dataset_id});
+    } else {
+      cy.log('Auth enabled, skip seed cleaning.');
```

Comment:
No remote connection to elasticsearch. Test remote instances will be discardable so we won't need to clean regardless.